app/views/reviews/_form.html.erb

```diff
@@ -0,0 +1,8 @@
+<%= bootstrap_form_for @review, url: playspace_reviews_path(@playspace.id) do |f| %>
+  <div>
+    <%= f.text_field :comment %>
+  </div>
+  <div>
```

Comment:
if you use `bootstrap_form_for` you don't need to have a `<div></div>` around your `f.text_field`